atroposlib/envs/server_handling/openai_server.py

```diff
@@ -49,6 +50,7 @@ async def check_server_status_task(self, chat_completion: bool = True):
                 self.server_healthy = False
             await asyncio.sleep(1)
 
+    @weave.op
```

Comment:
can you do it on the base server instead? That way we don't need to wrap in multiple places